lib/components/src/tooltip/ListItem.tsx

```diff
@@ -82,6 +81,18 @@ const Center = styled.span({
   },
 });
 
+const Column = styled.span({
```

Comment:
Awesome idea Jimmy! 

Can we do this without changing the tooltip API? That way the impact is constrained to only this tooltip.

For instance, `Title` renders a span that allows arbitrary nodes. Perhaps we pass the desc into the `title` like so:

```
title={
	<>
	Blurred vision
	<desc>21% of the population</desc>
	</>
}
```
and for the styling in `VisionSimulator.tsx`

```
const Description = styled.span({
 // add more specific styles here
})
```

Reply:
Hi @domyen 👋 
Yeah no problem, I can revert that.
Nice plan to remove the components in favor of components in DS BTW !